Dockerfile-dev

```diff
@@ -25,15 +25,6 @@ USER app
 
 WORKDIR /home/app/src
 
-# Copy files needed for installing gem dependencies, and install them.
-COPY Gemfile Gemfile.lock ./
-COPY plugins ./plugins
-RUN bundle config build.nokogiri "--use-system-libraries" && \
```

Comment:
does `--use-system-libraries` still have any use, e.g. not having to compile libxml and friends at `bundle install` time?